ui/templates/importer/upload.html

```diff
@@ -0,0 +1,9 @@
+{% extends "lore/base.html" %}
```

Comment:
I think just base.html should work, and would be preferred, but happy to be wrong if that is wrong...